yeastphenome/apps/conditions/models.py

```diff
@@ -114,9 +108,7 @@ def phenotypes(self):
         ).distinct()
 
     def phenotypes_link_list(self):
```

Comment:
Is this function used anymore in the admin (should it be added?) I see it defined in papers and conditions models.py, but not used for either when I grep for it.

Reply:
I will have  a look now at these functions and update this pull request as necessary.